v0/destinations/am/config.js

```diff
@@ -113,6 +113,11 @@ const BATCH_EVENT_ENDPOINT = "https://api.amplitude.com/batch";
 const GROUP_ENDPOINT = "https://api.amplitude.com/groupidentify";
 const ALIAS_ENDPOINT = "https://api.amplitude.com/usermap";
 
+const EU_ENDPOINT = "https://api.eu.amplitude.com/2/httpapi";
+const EU_BATCH_EVENT_ENDPOINT = "https://api.eu.amplitude.com/batch";
+const EU_ALIAS_ENDPOINT = "https://api.eu.amplitude.com/usermap";
+const EU_GROUP_ENDPOINT = "https://api.eu.amplitude.com/groupidentify";
```

Comment:
EU endpoint for [identify](https://developers.amplitude.com/docs/identify-api#endpoint) is missing.